src/core/blaze.py

```
@@ -6,14 +6,12 @@
 import os, multiprocessing, json, time, logging, hashlib, signal, sys
 from datetime import datetime
 import croniter
-from .logger import BlazeLogger
+from .jobs import BlazeJobs
```

Ruff (0.12.2)

`.jobs.BlazeJobs` imported but unused

Remove unused import: `.jobs.BlazeJobs`

(F401)